packages/cascara/src/ui/Table/fixtures/ActionStack.Table.fixture.js

```diff
@@ -178,4 +178,4 @@ class Fixture extends PureComponent {
   }
 }
 
-export default Fixture;
+export default <Fixture />;
```

Comment:
Changing these fixtures to JSX elements exposes their props (and state for class components) in Cosmos.